__tests__/server/webservices/commuter.js

```diff
@@ -13,9 +13,9 @@ describe('commuter', () => {
 
   const initCommuterData = {
     address: '123 Main St',
-    location: {
+    coordinate: {
       lat: 12,
-      lon: 34
+      lng: 34
```

Comment:
We should definitely have this in the first format. Let's have everything be represented in `{lon, lat}`.